src/components/ColorsGrid/utils.ts

```diff
@@ -0,0 +1,45 @@
+const parseRgba = (color: string) => {
+    return color
+        .slice(color.indexOf('(') + 1, -1)
+        .split(', ')
+        .map(Number);
+};
+
+const getComputedColor = (color: string) => {
+    const element = document.createElement('div');
+
+    element.style.color = color;
+    document.body.appendChild(element);
+
+    const computedColor = window.getComputedStyle(element).color;
```

Comment:
Why we can't use elements from the component itself?

Reply:
It is necessary to calculate the color before the first display of the elements. Doing this in useEffect after the first render doesn't seem like a good idea